tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -256,6 +256,14 @@ await VerifyHelper.VerifySpans(spans, settings)
             }
         }
 
+        protected static void FilterConnectionHeader(VerifySettings settings)
+        {
+            Regex appSecConnectionHeader0 = new(@"_dd.appsec.fp.http.header: hdr-0\d", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
So just to clarify, this regex simply sets the _second_ digit of the regex to `X`, correct? That's the only expected change?